vue-components/src/components/Button.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<button :class="[ 'wikit', 'wikit-Button', `wikit-Button--${ flavor }` ]">
+		<slot />
+	</button>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * A Button
+ */
+export default Vue.extend( {
+	name: 'Button',
+	props: {
+		/**
+		 * The flavor of the button.
+		 * One of: neutral
+		 */
+		flavor: {
+			type: String,
+			validator( value: string ): boolean {
+				return [ 'neutral' ].includes( value );
+			},
+			default: 'neutral',
+		},
+	},
+} );
+</script>
+
+<style lang="scss">
+/**
+ * Construction site identified:
+ * How do we name the component level token?
+ *
+ * Suggestion for naming scheme:
+ * wikit-<Component>-[prop]-[state]-css-property
+ */
+
+.wikit-Button {
+	padding-inline: $wikit-Button-medium-padding-horizontal;
+	padding-block: $wikit-Button-medium-padding-horizontal;
```

Comment:
We were really struggling with telling apart horizontal and vertical...

Reply:
Interesting! did you agree on a more familiar name? These tokens were previously named "-padding-x & "-padding-y" respectively, but I found that could be confusing too

Reply:
My comment was strictly about us doing a mistake when mapping both "inline" (x) and "block" (y) to "horizontal" (instead of "vertical" for "block").

As the tokens we are seeing are component-level tokens and created within the same commit they were never called "x" or "y". Maybe they were called like this in the aliases or globals but this is not what this is about - but both is fine in my eyes.